daemons/fenced/fenced_commands.c

```diff
@@ -323,6 +326,23 @@ fork_cb(GPid pid, gpointer user_data)
     cmd->activating_on = NULL;
 }
 
+static int
+get_agent_metadata_cb(gpointer data) {
+    stonith_device_t *device = data;
+
+    device->agent_metadata = get_agent_metadata(device->agent);
+    if (device->agent_metadata) {
+        read_action_metadata(device);
+        stonith__device_parameter_flags(&(device->flags), device->id,
+                                        device->agent_metadata);
+        return G_SOURCE_REMOVE;
+    } else {
+        if (device->timer->period_ms < 160 * 1000)
+            mainloop_timer_set_period(device->timer, 2 * device->timer->period_ms);
```

Comment:
Style is to always use { } ... besides being visually more obvious, it makes diffs easier to review later if we need to add a line inside